netpoll/handle.go

```diff
@@ -16,20 +16,31 @@ type filer interface {
 type Desc struct {
 	file  *os.File
 	event Event
+	fd    int
 }
 
 // NewDesc creates descriptor from custom fd.
 func NewDesc(fd uintptr, ev Event) *Desc {
-	return &Desc{os.NewFile(fd, ""), ev}
+	return &Desc{file: os.NewFile(fd, ""), event: ev, fd: int(fd)}
+}
+
+// NewDescFile creates descriptor from os.File
```

Comment:
Hi! Could you please follow [this rules](https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences) for comments? Thanks!

Reply:
fixed.